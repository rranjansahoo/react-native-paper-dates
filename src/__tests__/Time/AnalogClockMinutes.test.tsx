import * as React from 'react'

import renderer from 'react-test-renderer'
import AnalogClockMinutes from '../../Time/AnalogClockMinutes'

it('renders AnalogClockMinutes', () => {
  const tree = renderer.create(<AnalogClockMinutes minutes={45} />).toJSON()

  expect(tree).toMatchSnapshot()
})
