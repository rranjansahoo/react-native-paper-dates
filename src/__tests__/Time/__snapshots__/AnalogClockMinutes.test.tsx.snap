// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AnalogClockMinutes 1`] = `
[
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 128,
          "top": 26,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        00
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 179,
          "top": 39,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        5
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 217,
          "top": 77,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        10
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 230,
          "top": 128,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        15
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 217,
          "top": 179,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        20
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 179,
          "top": 217,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        25
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 128,
          "top": 230,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        30
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 77,
          "top": 217,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        35
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 39,
          "top": 179,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        40
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 26,
          "top": 128,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              {
                "color": "rgba(255, 255, 255, 1)",
              },
            ],
          ]
        }
      >
        45
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 39,
          "top": 77,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        50
      </Text>
    </View>
  </View>,
  <View
    pointerEvents="none"
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 24,
          "height": 48,
          "justifyContent": "center",
          "marginLeft": -24,
          "marginTop": -24,
          "position": "absolute",
          "width": 48,
          "zIndex": 20,
        },
        {
          "left": 77,
          "top": 39,
        },
      ]
    }
  >
    <View
      style={
        {
          "borderRadius": 24,
        }
      }
    >
      <Text
        maxFontSizeMultiplier={1.5}
        selectable={false}
        style={
          [
            {
              "textAlign": "left",
            },
            {
              "color": "rgba(28, 27, 31, 1)",
              "writingDirection": "ltr",
            },
            [
              {
                "fontFamily": "System",
                "fontSize": 16,
                "fontWeight": "400",
                "letterSpacing": 0.15,
                "lineHeight": 24,
              },
              undefined,
            ],
          ]
        }
      >
        55
      </Text>
    </View>
  </View>,
]
`;
